modules/cli/src/main/scala/scala/cli/commands/run/Run.scala

```diff
@@ -113,13 +113,22 @@ object Run extends ScalaCommand[RunOptions] with BuildCommandHelpers {
   }
 
   def runCommand(
-    options: RunOptions,
+    options0: RunOptions,
```

Comment:
why the rename?

Reply:
Because I'm defining `options` in line `126`, where server may be set to false. It felt like to easiest way to rename it like this. Would you rather it was named differently?

Reply:
ah, I see, missed that.
Nah, it's fine. I'm not a fan of this convention, but we do use it in the codebase in many spots.